app/desktop/studio_server/tool_servers_api.py

```diff
@@ -21,6 +23,10 @@ class ExternalToolServerCreationRequest(BaseModel):
     description: str | None = None
 
 
+class ExternalToolServerApiDescription(ExternalToolServer):
```

Comment:
I'd remove the inheritance. You're getting fields you don't want, like parent. Just define your API here. It might be similar, but it's not really inheritance.